pkg/apis/paddlepaddle/v1/register.go

```diff
@@ -0,0 +1,32 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	// SchemeBuilder will call register
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// AddToScheme will apply all the stored functions to the scheme
+	AddToScheme = SchemeBuilder.AddToScheme
+)
+
+// SchemeGroupVersion is the group version used to register these objects.
+var SchemeGroupVersion = schema.GroupVersion{Group: CRDGroup, Version: CRDVersion}
+
+// Resource takes an unqualified resource and returns a Group-qualified GroupResource.
+func Resource(resource string) schema.GroupResource {
+	return SchemeGroupVersion.WithResource(resource).GroupResource()
+}
+
+// addKnownTypes adds the set of types defined in this package to the supplied scheme.
+func addKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
Maybe don't return `error` if it's not possible to have error. Otherwise the caller need to unnecessarily handle it.

Reply:
Because the parameter of the caller function `NewSchemeBuilder` is `funcs ...func(*Scheme) error`, we must return `nil` even if there is no error.